config/config.go

```diff
@@ -24,10 +32,17 @@ func Get() (*Config, error) {
 	}
 
 	cfg = &Config{
-		BindAddr:                   "localhost:26200",
+		BindAddr:                   "localhost:26300",
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		KafkaAddr:                  []string{"localhost:9092"},
+		KafkaVersion:               "1.0.2",
+		KafkaOffsetOldest:          true,
+		KafkaNumWorkers:            1,
+		HelloCalledGroup:           "dp-import-cantabular-dimension-options",
```

Comment:
Can you have a look to see what other apps that use Kafka have renamed their 'HelloCalledGroup' and similar to something more appropriate ?
As this seems not quite right to me.

Reply:
For the very first PR we left the boilerplate as is with the last service, the update for the proper names being in the first skeleton update

Reply:
OK